industrial_ci/src/tests/clang_format_check.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright (c) 2018, Jonathan Hechtbauer
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function run_clang_format_check() {
+  local err=0
+  local path=$TARGET_REPO_PATH
+
+  ici_require_run_in_docker # this script must be run in docker
```

Comment:
I think it would make sense to use the base image for the format test, I will try it later..